src/Knp/DictionaryBundle/Validator/Constraints/Dictionary.php

```diff
@@ -16,8 +16,6 @@ final class Dictionary extends Constraint
 {
     /**
      * @var string
-     *
-     * @todo switch to native typehint after Symfony 4.4 compatibility was removed
```

Comment:
Did you miss to add the native typehint ?

Reply:
Oh, it's a new feature of 7.4.... In my mind it was a feature of 8.0